js-extensions/EXTENSIONS.md

```diff
@@ -51,12 +51,18 @@ Extensions are defined in a `jenkins-js-extensions.yaml` file in the javascript
 Properties are:
 - `component`: a module from which the default export will be used
 - `extensionPoint`: the extension point name
-- `type`: an optional data type this extension handles
+- `dataType`: an optional data type this extension handles
 
 For example, the `AboutNavLink` might be defined as a default export:
 
     export default class NavLink extends React.Component {
         ...
     }
 
+#### Enforcing specific component types
+
+In order to ensure a specific component is returned, an extension point may also use the `componentType` filter - it accepts an object prototype (e.g. an ES6 class), e.g.:
+
+    <Extensions.Renderer extensionPoint="test-view" componentType={TestResults} ... />
```

Comment:
@kzantow Is the difference explained somewhere? When would you use one over the other? When would you use both?